README.md

```diff
@@ -15,7 +15,7 @@ import { pushStream } from "dynamodb-stream-algolia";
 
 pushStream({
     event,
-    index: INDEX_NAME,
+    index_name: INDEX_NAME,
```

Comment:
change it back to index, everything else looks fine

Reply:
but It'll always create a new index in algolia with the name of the dynamo db table cause is the default in index.js file, am I right?

in idex.js:
```javascript
 exports.pushStream = async ({
  event,
  index_name = getTableNameFromARN(event.Records[0].eventSourceARN),
  appId,
  adminKey,
  logs = false,
} = {}) => {
  validateString(index_name, `index_name`)
  validateString(appId, `appId`)
  validateString(adminKey, `adminKey`)
```

can you see that you call it with index, and the function expect for a parameter called "index_name"?